src/server/auth/auth-helpers.ts

```diff
@@ -126,31 +128,37 @@ export function decodeToken<T extends Record<string, string>>(
   return jose.decodeJwt(jwtToken) as unknown as T;
 }
 
+function isIDPSessionExpired(expiresAt: string) {
+  return new Date(parseInt(expiresAt, 10) * ONE_SECOND_MS) < new Date();
```

Comment:
Dit lijkt op een seconden naar miliseconden conversie. Maar misschien kunnen we de unit aan de naam of type met een doc comment toevoegen.